packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1626,7 +1626,11 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
           // platforms, and only then if the physics allow it.
           break;
       }
-      position.jumpTo(newPosition);
+      if (_thumbDragging) {
+        position.dragTo(newPosition);
+      } else {
+        position.jumpTo(newPosition);
```

Comment:
If `dragTo` is just calling `jumpTo`, what is the difference?

Reply:
dragTo captures a specific use gesture. It gives the scrollbar a way to distinguish a thumb-drag, which is something that the user decided to do, vs a programmatic scroll, which might not be. The idea was to give the app a way to change it's behavior only while the thumb is being dragged, rather than any time the app jumped the scroll offset around.